jnigen/test/jackson_core_test/jnigen.yaml

```diff
@@ -1,13 +1,13 @@
 maven_downloads:
   source_deps:
     - 'com.fasterxml.jackson.core:jackson-core:2.13.4'
-  source_dir: test/jackson_core_test/third_party/java
```

Comment:
Now values in jnigen are resolved relative to YAML file. See `getPath` and `getPathList` in yaml_reader.dart

Reply:
Yeah that's better. We did a similar change in ffigen recently.